modules/apps/object/object-test/src/testFunctional/tests/ObjectFieldsDateAndTime.testcase

```diff
@@ -64,5 +64,90 @@ definition {
 		}
 	}
 
+	@description = "LPS-143068  - Verify if is possible to see DateTime with differents Time Zone"
+	@priority = 5
+	test CanSeeDifferentTimeZone {
+		property portal.acceptance = "true";
+
+		task ("Given a custom object with a date and time field as UTC") {
+			ObjectAdmin.addObjectViaAPI(
+				labelName = "Custom Object 186248",
+				objectName = "CustomObject186248",
+				pluralLabelName = "Custom Objects 186248");
+
+			ObjectAdmin.publishObjectViaAPI(objectName = "CustomObject186248");
+
+			ObjectAdmin.openObjectAdmin();
+
+			ObjectPortlet.selectCustomObject(label = "Custom Object 186248");
+
+			ObjectAdmin.goToFieldsTab();
+
+			ObjectAdmin.addObjectFieldViaUI(
+				fieldLabel = "Custom Date Time",
+				fieldTimeStorage = "Convert to UTC",
+				fieldType = "Date and Time");
+
+			ObjectAdmin.publishObjectViaAPI(objectName = "CustomObject186248");
+
+			User.openUsersAdmin();
+
+			UserNavigator.gotoUser(userScreenName = "test");
+
+			UserNavigator.gotoDisplaySettings();
+
+			Select(
+				locator1 = "Select#TIME_ZONE",
+				value1 = "(UTC) Coordinated Universal Time");
+
+			PortletEntry.save();
+		}
+
+		task ("And given an object entry") {
+			ObjectAdmin.addObjectSingleFieldEntryViaAPI(
+				fieldName = "customDateTime",
+				objectName = "CustomObject186248",
+				value = "2023-06-01T12:00:00.000Z");
+		}
+
+		task ("And given a user with different time zone") {
+			JSONUser.addUser(
+				userEmailAddress = "userea@liferay.com",
+				userFirstName = "userfn",
+				userLastName = "userln",
+				userScreenName = "usersn");
+
+			JSONRole.assignRoleToUser(
+				roleTitle = "Administrator",
+				userEmailAddress = "userea@liferay.com");
+		}
+
+		task ("When access the user") {
+			User.logoutAndLoginPG(
+				userLoginEmailAddress = "userea@liferay.com",
+				userLoginFullName = "userfn userln");
+
+			User.openUsersAdmin();
+
+			UserNavigator.gotoUser(userScreenName = "usersn");
+
+			UserNavigator.gotoDisplaySettings();
+
+			Select(
+				locator1 = "Select#TIME_ZONE",
+				value1 = "(UTC -03:00) Brasilia Time");
+
+			PortletEntry.save();
```

Comment:
Do we need to do this as `usersn` or can we set the the time zone for the user as `Test Test`?